### ISSUE TITLE
Coding Exercise 2 in W1D3 Tutorial 2

### ISSUE THREAD

**Issue by HaiyangJin** [NeuromatchAcademy/course-content-dl#212]
In Coding Exercise 2: Dataloader on a real-world dataset, the `return` for the function seems missing. 

Specifically, probably `return img_train_loader, img_test_loader` should be added for function `get_data_loaders(batch_size, seed)`.

**Comment by spirosChv** [comment]
cood catch. the solution is correct but somehow in the student version, there was no `return` for the function. Thanks
